src/Job/DeveloperSync.php

```diff
@@ -130,21 +131,42 @@ protected function executeRequest() {
   /**
    * {@inheritdoc}
    */
-  public function execute() : bool {
+  public function execute(): bool {
     parent::execute();
 
+    // Update Apigee Edge developers and Drupal users if needed.
+    $identical_accounts = array_intersect($this->edgeAccounts, $this->drupalAccounts);
+    foreach ($identical_accounts as $search => $mail) {
+      /** @var \Apigee\Edge\Api\Management\Entity\DeveloperInterface $developer */
+      $developer = Developer::load($mail);
```

Comment:
I'd probably load all developers by email outside of this scope instead of loading them one by one in a loop.